src/program/services/downloaders/__init__.py

```
@@ -63,7 +63,7 @@ def run(self, item: MediaItem):
                     download_success = True
                     break
                 else:
-                    raise NoMatchingFilesException(f"No valid files found for stream {stream.infohash}")
+                    raise NoMatchingFilesException(f"No valid files found")
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)